code/modules/client/client procs.dm

```diff
@@ -414,9 +414,16 @@
 		if(M.client)
 			playercount += 1
 
-	if(playercount >= 150 && GLOB.panic_bunker_enabled == 0)
-		GLOB.panic_bunker_enabled = 1
-		message_admins("Panic bunker has been automatically enabled due to playercount surpassing 150")
+	// Update the state of the panic bunker based on current playercount
+	var/threshold = config.panic_bunker_threshold
+
+	if((playercount > threshold) && (GLOB.panic_bunker_enabled == FALSE))
```

Comment:
Can write the == FALSE as `!GLOB.panic_bunker_enabled`

Reply:
I prefer the explicit use of `== FALSE` in the comparison expression.
I won't change it, but I wouldn't be upset if a maintainer chose to do so before merging.